pkg/rke/templates/templates.go

```diff
@@ -213,12 +214,20 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 			">=1.8.0-rancher0 <1.15.0-rancher0":       flannelv18,
 		},
 		kdm.CoreDNS: {
-			">=1.21.9-rancher1-2":                     coreDnsv183Rancher2,
-			">=1.21.0-rancher1-1 <1.21.9-rancher1-2":  coreDnsv183,
-			">=1.20.15-rancher1-2 <1.21.0-rancher1-1": coreDnsv183Rancher2,
-			">=1.17.0-alpha <1.20.15-rancher1-2":      coreDnsv117,
-			">=1.16.0-alpha <1.17.0-alpha":            coreDnsv116,
-			">=1.8.0-rancher0 <1.16.0-alpha":          coreDnsv18,
+			// the 1.28.x line
+			">1.28.8-rancher1-1":                      coreDnsv183Rancher3,
+			">=1.28.0-rancher1-1 <=1.28.8-rancher1-1": coreDnsv183Rancher2,
+			// the 1.27.x line
+			">1.27.12-rancher1-1 <1.28.0-rancher1-1":   coreDnsv183Rancher3,
+			">=1.27.0-rancher1-1 <=1.27.12-rancher1-1": coreDnsv183Rancher2,
+			// the 1.26.x line
+			">1.26.15-rancher1-1 <1.27.0-rancher1-1":   coreDnsv183Rancher3,
```

Comment:
Note:
`v1.26.15-rancher-1-1`, `v1.27.12-rancher-1-1`, and `v1.28.8-rancher-1-1` are the versions that will be released in the KDM OOB March patch release before this PR is merged, so the new template should be applied to any version that is higher than them in each release line.